store/keystore.go

```diff
@@ -199,3 +208,41 @@ func saveToPem(out io.Writer, key openssl.PrivateKey) error {
 
 	return err
 }
+
+func loadPassphrase(passphrase_file string) (passphrase string, err error){
+	if passphrase_file != "" {
+		if passphrase_file == "-" {
+			log.Debugf("Read passphrase from pipe.")
+			fi, err := os.Stdin.Stat()
+			if err != nil {
+				log.Fatal(err)
+				return "", err
+			}
+			if fi.Mode() & os.ModeNamedPipe == 0 {
+			} else {
+				reader := bufio.NewReader(os.Stdin)
+				var output []rune
+				for {
+					input, _, err := reader.ReadRune()
+					if err != nil && err == io.EOF {
+						break
+					}
+					output = append(output, input)
+				}
+				passphrase = string(output[:len(output)-1])
+			}
+		} else {
+			log.Debugf("Read passphrase from file.")
+			dat, err := ioutil.ReadFile(passphrase_file)
+			if err != nil {
+				log.Fatal(err)
+				return "", err
+		    }
+			passphrase = string(dat[:len(dat)-1])
```

Comment:
I assume you did this because of the trailing newline many files have. However, I believe it's better to do:

```
passphrase = strings.TrimRight(string(dat), "\n")
```

This will work also if the file does not have a trailing newline.

Reply:
Checked that and your version also works. Will change this in the next commit.